ex12/main.go

```diff
@@ -1,5 +1,16 @@
 package main
 
-  func main() {
+import "fmt"
 
-  }
+type void struct{}
+
+func main() {
+	a := []string{"aboba", "abeba", "aboba", "abeba", "abeba"}
+	set := make(map[string]void)
```

Comment:
We don't need to define the empty struct to use it
```suggestion
	set := make(map[string]struct{})
```